settings.gradle.kts

```diff
@@ -6,7 +6,7 @@ pluginManagement {
 	}
 }
 dependencyResolutionManagement {
-	repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+	repositoriesMode.set(RepositoriesMode.PREFER_PROJECT)
```

Comment:
1. https://stackoverflow.com/a/75032841/10894077
2. https://slack-chats.kotlinlang.org/t/8141270/hey-all-so-i-ve-had-this-issue-happen-on-both-a-personal-pro

Reply:
It's mentioned that PREFER_PROJECT is the default one anyway, can we remove this line completely?

Reply:
Indeed, and I was thinking which way would be better. At first glance, it's not clear why setting `FAIL_ON_PROJECT_REPOS` shouldn't be here, because it's not present for empty KMP project created by IDE wizard.  Because of that I decided to set desired value to keep it more explicit. 
Of course, I'm not emotionally attached to, so we can remove it depending on your suggestion 😄

Reply:
For me always less code is better 😄 but I'll leave it up to you, it's not a big deal to keep it as well

Reply:
I'm also adhere to that rule, so I'll remove it to consistent 😄  
Thanks 🍻